### PR TITLE
Improve Spanish translations

### PR DESCRIPTION
**Pull request by cpinamtz**
As a Spanish native speaker, there are some sentences that were wrongly translated.

### CODE REVIEW

**Comment by DRracer** [comment]
@cpinamtz thank you for your PR. It's good to see, that the comunity cares about the translations as well.
Could you please do the following:
- checkout the current master and look at the most recent Spanish translation PO file?
- fix it and update your PR here accordingly
- try building the FW with your changes

**Please note** - the FW build now finally includes a full-featured build script for the languages from the PO files directly - so when you build your FW and upload to your MINI, you should see your changes. Please let us know if that works for you, this build system improvement is relatively new.

**Comment by eried** [comment]
Hola @cpinamtz 

Algunos comentarios, ¿es natural llamarle fusor? ¿no sería mejor llamarle extrusor? ej.:

`msgstr "Comprobando el funcionamiento del extrusor y de la base"`

Lo mismo con "PURGAR", crees que sonaría mejor "Extruir"?

**Comment by cpinamtz** [comment]
@DRracer I've rebased new changes and update my PR accordingly. Also, I've used the polite mode of the verb (it's like when you're referring to Mr. or Mrs.). I've run the `python utils/build.py` command and everything has been successfully. Unfortunately, I can't test it with my printer because I've not removed the appendix yet. It would be nice if someone can test it to see if everything works fine.

Sí, es más conocido "extrusor" que "fusor", la verdad. De la misma forma que cuando se habla de la "base". No es incorrecto, pero al menos aquí en España está más extendida la palabra "cama". Respecto a la duda que comentas entre "purgar" y "extruir" son dos cosas diferentes. Purgar implica extruir (en este caso, se extruye lo que queda del filamento antiguo para asegurarse de que no queda nada y se extruye algo del nuevo) pero extruir no implica purgar. Por lo tanto, ahí considero que está bien la diferenciación.

**Comment by DRracer** [comment]
@cpinamtz thank you for rebasing your original PR and reviewing the ES translation. 
Also I'm very happy the build system works for you! No need to upload it into MINI if you still have the appendix intact. The information alone, that the build system works is enough.

Our ES translator will go through your changes soon and we'll decide if this PR will get merged as it is or if we change the translation in our system and export a new PO file instead.

Anyway, your work will not be forgotten ;) .

**Comment by cpinamtz** [comment]
@DRracer I'm glad to read that! Also, I'd like to contribute to the [Spanish assembly manual](https://www.prusa3d.com/downloads/manual/prusa3d_manual_mini_es.pdf) 'cause I found some wrong translations, how can I do it?

**Comment by DRracer** [comment]
@cpinamtz FYI - your changes will to be implemented into PR #1104 within this week.
I'll pass your offer regarding the manual to our Content team, thanks for your offer

File: src/lang/po/es/Prusa-Firmware-Buddy_es.po [hunk 1]
```diff
@@ -116,41 +116,41 @@ msgstr "Cambiar Filamento"
...
 msgstr ""
-"Revisa el cableado del termistor de la base calefactada buscando posibles "
+"Revise el cableado del termistor de la base calefactada para detectar posibles "
 "daños."
 
 #: src/common/bsod.cpp:234
 msgid "Check the print head heater & thermistor wiring for possible damage."
 msgstr ""
-"Revisa el cableado del termistor y del calentador del fusor para ver si hay "
+"Revise el cableado del termistor y del calentador del extrusor para ver si hay "
```
Comment (misan) [inline src/lang/po/es/Prusa-Firmware-Buddy_es.po:131]: You are wrongly assuming hotend==extruder. The extruder is the mechanism that pushes filament into the hotend. Hotend is where the filament is melted and deposited through the nozzle.

Comment (eried) [reply src/lang/po/es/Prusa-Firmware-Buddy_es.po:131]: I also think it is weird to call it fusor but in any case the perfect translation would be "punta caliente"

**Comment by DRracer** [comment]
@cpinamtz thank you for your pull request, our translators went through your proposal and picked relevant improvements into our FW - PR #1112. Please note, the messages in our FW are meant to be more a casual talk. Closing this PR.

**Pull request closed without merging**